content/en/blog/_posts/2021-04-29-SIG-Usability-Spotlight.md

```diff
@@ -0,0 +1,66 @@
+
+Accessibility guidelines for Kubernetes community artifacts, examples include internationalization of documentations, color choices for people with color blindness, ensuring compatibility with screen reader technology, user interface design for core components with user interfaces, and more.
+
+### Q. What do you do to help new contributors get started?
+
+A. New contributors can get started by shadowing one of the user interviews, going through user interview transcripts, analyzing user interviews, and designing surveys.
+
+SIG Usability is also open to new project ideas. If you have an idea, we’ll do what we can to fulfill that idea. There are also regular SIG Meetings for people to attend and ask their questions live. The meetings are also recorded for those who may not be able to attend. As always, you can reach out to us on Slack as well.
+
+### Q. What does the survey include?
+
+A. In simple terms, the survey gathers information about how people use Kubernetes, such as trends in learning to deploy a new system, error messages they receive, and workflows.
+
+One of our goals is to standardise the responses accordingly. The ultimate goal is to analyze survey responses for important user stories whose needs aren't being met.
+
+### Q. Are there any particular skills you’d like to recruit for? What skills are contributors to SIG Usability likely to learn?
+
+A. Although contributing to SIG Usability does not have any pre-requisites as such, but experience with user research, qualitative research, prior experience with how to conduct an interview would be a great plus point. Quantitative research, like survey design and screening, is also helpful and something that we expect contributors to learn.
+
+### Q. What are you getting positive feedback on, and what’s coming up next for SIG Usability?
+
+A. We have had new members joining and coming to monthly meetings regularly and showing interests in becoming a contributor and helping the community. We have also had a lot of people reach out to us via Slack showcasing their interest in the sig. 
+
+Currently, we are focused on finishing this study, which is our project for this year. We are always happy to have new contributors join us.
+
+### Q: Any closing thoughts/resources you’d like to share?
+
+A. We love meeting new contributors and perceive how we can assist them with investigating different spaces of the Kubernetes project. We will work with and team up with other SIGs on giving end-user channels for demonstrating input on their experience, studies, and other criticism get-togethers around project convenience, documentation, plan components, and language rules for Kubernetes concepts, recognizing opportunities for user research and best strategies to accomplish those.
```

Comment:
```suggestion
A. We love meeting new contributors and assisting them in investigating different Kubernetes project spaces. We will work with and team up with other SIGs to facilitate engaging with end-users, running studies, and help them integrate accessible design practices into their development practices.
```

The 2nd sentence didn't make much sense as read, I think this suggestion covers the intent of it? :x